client/app/views/AllergensView.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { StyleSheet, View, TouchableOpacity, Text, ScrollView } from 'react-native';
+import { Transition } from 'react-navigation-fluid-transitions';
+
+import connectToRedux from '../redux/lib/connectToRedux';
+import sp from '../redux/lib/stateProperties';
+
+import Header from '../components/Header';
+import Allergen from '../components/Allergen';
+import BottomTabs from '../components/BottomTabs';
+
+import styles from '../config/styles';
+
+class AllergensView extends Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    componentDidMount() {
+        // TODO: make sure that this is the appropriate interface to get the itemID
+        this.props.getMenuItemInformation(this.props.menu.data.itemID);
```

Comment:
How can I avoid making the API call every time a new tab is pressed? Ideally, we want to call once, and then just pull from the store every time the tab changes. Right now, we call every time the tab changes.